lib/mergometer/report.rb

```diff
@@ -1,12 +1,16 @@
 require "progress_bar"
 require "mergometer/pull_request"
 
+reports_path = File.expand_path("./reports/**/*.rb", __FILE__)
+Dir[reports_path].each { |f| require f }
+
 module Mergometer
   class Report
     require "hirb"
 
-    def initialize(repo)
+    def initialize(repo, from = nil)
       @repo = repo
+      @from = from || (Date.today - 1.day).beginning_of_week.strftime("%Y-%m-%d")
```

Comment:
This is a little hard for me to parse 🙇  when is this? Yesterday, but the beginning of the week?

Reply:
Yeah, I thought if we run script on sunday, it should get prs from last week rather than only for sunday 🤔